configure.ac

```diff
@@ -2243,8 +2243,11 @@ dnl struct {char *name; int *addr; char *doc;} desc = spam(foo, "something");
 dnl ]], [[;]])],[cpp_type=ansi],[AC_DEFINE(HAVE_OLD_CPP) cpp_type=traditional])
 dnl AC_MSG_RESULT($cpp_type)
 
+dnl backwards compatibility: autoconf 2.71 deprecates STDC_HEADERS
+dnl assume C99 compilers provide ANSI C headers
```

Comment:
Regarding C99: What about checking `AC_PROG_CC_C99` early on and bail if `ac_cv_prog_cc_c99` is `no`?

Reply:
I like the idea